src/app.service.ts

```diff
@@ -1,8 +1,62 @@
 import { Injectable } from '@nestjs/common';
+import { LogMessageFormat } from "logging-format";
+import { AlertConverterService } from './alert-converter/alert-converter.service';
+const { Kafka } = require('kafkajs');
+
+const kafka = new Kafka({
+  clientId: 'prometheus-alert-converter',
+  brokers: ['localhost:9092']
+});
+const producer = kafka.producer();
 
 @Injectable()
 export class AppService {
-  getHello(): string {
-    return 'Hello World!';
+
+  constructor(
+    private alertConverter: AlertConverterService,
+  ) { }
+
+  /**
+   * Takes Alert in correct JSON Format and converts into LogMessage Array.
+   * @param alert the Alert in the Format: https://prometheus.io/docs/alerting/latest/configuration/#webhook_config
+   */
+  convertAlertToLogMessages(alert): LogMessageFormat[] {  
+
+    return this.alertConverter.alertToLogMessages(alert); ;
+
+  }
+
+  /**
+   * Sends the log message to the kafka topic logs
```

Comment:
markiert, das 'logs' der name des topics ist. dann liest sich der satz viel flüssiger.
```suggestion
   * Sends the log message to the kafka topic 'logs'
```